modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -69,6 +70,60 @@ update the generated code for that resource type:
 $ operator-sdk generate k8s
 ----
 
+. *Optional: Add custom validation to your CRD.*
++
+OpenAPIv3 schemas are added to CRD manifests in the `spec.validation` block when
```

Comment:
Under "Additional resources" you have `OpenAPI v3.0`, which I believe would be the preferred formatting. Any reason it's `OpenAPIv3` in this step>